docs/Documentation.md

```diff
@@ -1,29 +1,39 @@
----
-layout: page
-title: Documentation guide
----
+<frontmatter>
+  layout: default.md
+  title: "Documentation guide"
+  pageNav: 3
+  pageNavTitle: "Table Of Content"
+</frontmatter>
+
+# Documentation guide
 
 **Setting up and maintaining the project website:**
 
-* We use [**Jekyll**](https://jekyllrb.com/) to manage documentation.
+* We use [**MarkBind**](https://markbind.org/) to manage documentation.
 * The `docs/` folder is used for documentation.
-* To learn how set it up and maintain the project website, follow the guide [_[se-edu/guides] **Using Jekyll for project documentation**_](https://se-education.org/guides/tutorials/jekyll.html).
+* To learn how set it up and maintain the project website, follow the [_**MarkBind User Guide**_](https://markbind.org/userGuide/gettingStarted.html).
+  * Quick start:
+    * `cd docs # go to the docs folder`
+    * `npm i # install dependencies, only needed once when setting up the project`
```

Comment:
How about `npm ci` instead?

Reply:
If `npm ci` is used, then having a pin on the v4 version instead of the specific v4.1.0 may not made sense https://github.com/se-edu/addressbook-level3/pull/156#discussion_r1103647490

Because npm ci will not update the version of markbind-cli. Thoughts?

Reply:
Right, then should we remove the `package-lock` file since that would get overwritten after `npm i` anyway?

Reply:
hmm, it will only be overridden when there are changes to the version downloaded? If we remove it, do you think we should gitignore the package-lock? I'm not sure if that's a common practice.

Reply:
I would actually prefer pegging the version to a specific release for stability, and hence use `npm ci`. Tagging @damithc for comment on this.

Reply:
I would also prefer to peg to a specific release as future releases may introduce bugs. Using `npm i` may be risky.

Reply:
will change to npm ci for now